functions/scheduled/events.js

```diff
@@ -29,19 +29,19 @@ module.exports.checkForEvents = async function () {
 
             const message = await this.generateMessage(event, parameters, timeDifference, isEventSoon, startTimeDate);
 
-            await slack_handler.postMessageToChannel((parameters.alert_type === "alert-main-channel" ? "<!channel>\n" : "") + message, parameters.main_channel);
+            await slack_handler.postMessageToChannel((parameters.alert_type === "alert-main-channel" ? "<!channel>\n" : "") + message, parameters.main_channe, true);
```

Comment:
Intentional change (main_channel to main_channe)?

Reply:
Unintentional, fixed. Good eye, definitely slipped past me